### ISSUE TITLE
Document full Oz profile JSON syntax

### ISSUE THREAD

**Issue by flamsmark** [subgraph/oz#114]
I can't find any comprehensive documentation for the syntax used for the JSON files in `/var/lib/oz/cells.d/`. This would be really useful for writing my own Oz configs.

**Comment by dma** [comment]
The Oz profile is documented here:

https://github.com/subgraph/sgos_handbook

The SGOS handbook should be considered authoritative documentation.

It's mostly complete, I think, at least so far as the major/more mature parameters go. I'll need to re-examine that to make sure it is the case, so I'll leave the issue open.

**Comment by flamsmark** [comment]
I agree that it'd be great if the specs for Oz profiles were in the handbook. But the documentation [there](https://github.com/subgraph/sgos_handbook/blob/master/05_features_and_advanced_usage_02_oz.md#creating-an-oz-profile) is not quite comprehensive. I've seen plenty of options in the default profiles which aren't documented there, and when trying to write my own it's been trial and error trying to work out what the options are and what they do.
